### ISSUE TITLE
Fix Spelling of "Recent Contributors" Permalink on Navbar

### ISSUE THREAD

**Issue by PeculiarE** [publiclab/community-toolbox#542]
Hi, this is a [first-timers-only issue](https://code.publiclab.org/#r=all). This means we've worked to make it more legible to folks who either **haven't contributed to our codebase before, or even folks who haven't contributed to open source before.**

If that's you, we're interested in helping you take the first step and can answer questions and help you out as you do. Note that we're especially interested in contributions from people from groups underrepresented in free and open source software!

We know that the process of creating a pull request is the biggest barrier for new contributors. This issue is for you 💝

If you have contributed before, **consider leaving this one for someone new** and looking through our general [help wanted](https://github.com/publiclab/plots2/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22) issues. Thanks!

### 🤔 **What you will need to know.**
Nothing. This issue is meant to welcome you to Open Source :) We are happy to walk you through the process.

### 📋 **Step by Step**
 
- [ ] 🙋 **Claim this issue**: Claim the issue by commenting. If someone else has claimed it, ask if they've opened a pull request already and if they're stuck -- maybe you can help them solve a problem or move it along!

<details>
<summary> <i>💡 Learn how to claim 🙋 </i> </summary>

### Claiming an issue

Unless the issue is marked as reserved for someone, you can just say "I'd like to try this!" and then you've claimed it - no need to wait for someone to assign it to you. Just be sure you link your pull request (PR) to this issue so we can see where your solution is.

And open one early if possible - even before you've completed it with additional commits - and others can help you figure out any issues you may face.
</details>

- [ ] 📝 **Update** the file [index.html](https://github.com/publiclab/community-toolbox/blob/main/index.html) in the community-toolbox repository (press the little pen Icon) and edit the line as shown below.

[See this page](https://code.publiclab.org/#r=all) for some help in taking your first steps!

Below is a "diff" showing in red (and a -) which lines to remove, and in green (and a +) which lines to add:

```diff
@@ -78,7 +78,7 @@
                <ul>
                    <li><a class="line" href="#candidate">Candidates</a></li>
                    <li><a class="line" href="#fto-author">FTO Authors</a></li>
-                  <li><a class="line" href="#recent-contributer">Recent contributors</a></li>
+                 <li><a class="line" href="#recent-contributors">Recent contributors</a></li>
                    <li><a class="line" href="#contributer">Contributors</a></li>
                    <li><a class="line" href="#lang-sec">Languages Used</a></li>
                </ul>
```

- [ ]  💾 **Commit your changes**

- [ ]  🔀 **Start a Pull Request.** There are two ways how you can start a pull request:

1. If you are familiar with the terminal or would like to learn it, [here is a great tutorial](https://app.egghead.io/playlists/how-to-contribute-to-an-open-source-project-on-github) on how to send a pull request using the terminal.

2. You can also [edit files directly in your browser](https://help.github.com/articles/editing-files-in-your-repository/) and open a pull request from there.

 - [ ] 🏁 **Done** Ask in comments for a review :)

### Please keep us updated
💬⏰ - We encourage contributors to be respectful to the community and provide an update **within a week** of claiming a first-timers-only issue. We're happy to keep it assigned to you as long as you need if you update us with a request for more time or help, but if we don't see any activity a week after you claim it we may reassign it to give someone else a chance. Thank you in advance!

If this happens to you, don't sweat it! Grab another open issue.

### Is someone else already working on this?
🔗- We encourage contributors to link to the original issue in their pull request so all users can easily see if someone's already started on it.

👥- **If someone seems stuck, offer them some help!** Otherwise, [take a look at some other issues you can help with](https://code.publiclab.org/#r=all). Thanks!

### 🤔❓ Questions?
Leave a comment below, and [reach out in our chatroom](https://publiclab.org/chat)!

**Comment by jywarren** [comment]
It looks super! Thank you! Noting the exact change is in the spelling from `contributer` to `contributors`. (it took me a second to notice!)

Great work!

**Comment by PeculiarE** [comment]
Thank you! @jywarren

**Comment by AnanyaGarg28** [comment]
Hi, I would like to work on this issue. Please assign it to me! I am proficient with HTML, CSS, Bootstrap, JS and Nodejs. I am new to open-source and I would love to work on this FTO. Thanks!

**Comment by PeculiarE** [comment]
Hello @AnanyaGarg28.....this issue has already been assigned to/reserved for @adeola-dev. You can head over here to request for an FTO issue to be assigned to you: https://github.com/publiclab/plots2/issues/10153. Thank you!

**Comment by TildaDares** [comment]
Reserved for @adeola-dev

**Comment by AnanyaGarg28** [comment]
Alright, thanks!

**Comment by adeola-dev** [comment]
Thanks so much, I would work on it right away.

**Comment by adeola-dev** [comment]
https://github.com/adeola-dev/community-toolbox/pull/1

**Comment by adeola-dev** [comment]
https://github.com/adeola-dev/community-toolbox/pulls

**Comment by adeola-dev** [comment]
https://github.com/publiclab/community-toolbox/pull/543

**Comment by adeola-dev** [comment]
I just opened a PR. Your reviews will be appreciated.

**Comment by adeola-dev** [comment]
Hello @TildaDares, it has been merged by @jywarren. What am I to do next?

**Comment by TildaDares** [comment]
@adeola-dev I’ve seen it. Thanks!

**Comment by TildaDares** [comment]
Resolved in #543

**Comment by TildaDares** [comment]
Once you've completed an FTO, we
encourage folks to try making a first-
timers-only issue for someone else! And to
begin digging more into the open issues,
asking for clarifications, and, in the case of
Infragram, sharing mockups and design
ideas for discussion.
You can also look out for issues with the
help wanted label.
